package-loader-rule.js

```diff
@@ -0,0 +1,19 @@
+const path = require('path');
+
+const packages = [
+    'core',
+    'apple-pay',
+    'payment-integration',
+];
+
+const getPackagesWithPath = packages.map(package => {
+    const packageSrcPath =  path.join(__dirname, `packages/${package}/src`);
+
+    return {
+        test: /\.[tj]s$/,
+        include: packageSrcPath,
```

Comment:
I'm thinking, couldn't we just adjust the rule matcher so it matches all the input paths under `packages/*/src` (see https://webpack.js.org/configuration/module/#condition)? That way we don't have to maintain a static list?

Reply:
Cool I will have a look at this later today.

Reply:
I had a quick look and seems seems a bit tricky since we use `__dirname` as a prefix, not sure if that causes an issue. 

Also I had another thought where the list can be useful is to create the alias entry in webpack config, so maybe we can keep this change as is? 

This is a temporary change since when we have a package generator that should add the entries automatically.